### PR TITLE
Adding vPC+ params

### PR DESCRIPTION
**Pull request by dcheriancisco**
1. Added 3 new vPC+ properties to vpc_domain
2. Made changes to few existing properties to use range_summarize() for vlan_ranges
3. refactored beaker test to use new format and added vPC+ tests
4. Added a new demo_vpc_plus.pp manifest file in examples to demo vPC+

### CODE REVIEW

**Comment by mikewiebe** [comment]
@dcheriancisco Do you have the test results across all supported platforms?

**Comment by dcheriancisco** [comment]
Yes Do you want me to attach it here?

**Comment by dcheriancisco** [comment]
N5K Results:
==========
Begin cisco_vpc_domain/test_vpc_domain.rb

TestCase :: cisco_vpc_domain
  
  * 
  --------
   * TestStep :: Setup for cisco_vpc_domain provider test
  
  Found Platform string: 'Nexus5548 Chassis', Alias to: 'n5k'
  #### This device is of type: n5k #####
  
  ------------------------------------------------------------
  Section 1. Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  1.2 Default Properties exclusive to N6K and N7K :: default_properties_n6k7k :: SKIP
  Platform type does not match testcase platform regexp: /n(6|7)k/
  
  1.3 Default Properties exclusive to N7K :: default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 2. Non Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  2.2 Non Default Properties exclusive to N6K and N7K :: non_default_properties_n6k7k :: SKIP
  Platform type does not match testcase platform regexp: /n(6|7)k/
  
  2.3 Non Default Properties exclusive to N7K :: non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 3. vPC+ Non Default Property Testing
  
  3.1 vPC+ Non Default Properties on N7K :: vpc_plus_non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  3.1 vPC+ Non Default Properties on N7K :: vpc_plus_non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  ------------------------------------------------------------
    SKIPPED TESTS SUMMARY
  ------------------------------------------------------------
  1.2 Default Properties exclusive to N6K and N7K :: SKIP
  1.3 Default Properties exclusive to N7K  :: SKIP
  2.2 Non Default Properties exclusive to N6K and N7K :: SKIP
  2.3 Non Default Properties exclusive to N7K :: SKIP
  3.1 vPC+ Non Default Properties on N7K   :: SKIP
  3.1 vPC+ Non Default Properties on N7K   :: SKIP
  
  
  TestCase :: cisco_vpc_domain :: SKIP
  Warning: cisco_vpc_domain/test_vpc_domain.rb skipped in 145.99 seconds
      Test Suite: tests @ 2016-03-04 09:25:51 -0800

      - Host Configuration Summary -


              - Test Case Summary for suite 'tests' -
       Total Suite Time: 145.99 seconds
      Average Test Time: 145.99 seconds
              Attempted: 1
                 Passed: 0
                 Failed: 0
                Errored: 0
                Skipped: 1
                Pending: 0
                  Total: 1

      - Specific Test Case Status -
        
Failed Tests Cases:
Errored Tests Cases:
Skipped Tests Cases:
    Test Case cisco_vpc_domain/test_vpc_domain.rb skip
Pending Tests Cases:

  No tests to run for suite 'post_suite'
  Cleanup: cleaning up after successful run
  Warning: ssh connection to n5k-agent has been terminated
  Warning: ssh connection to puppetmaster-dc1 has been terminated
Beaker completed successfully, thanks.

N6K Results:
===========
Begin cisco_vpc_domain/test_vpc_domain.rb

TestCase :: cisco_vpc_domain
  
  * 
  --------
   * TestStep :: Setup for cisco_vpc_domain provider test
  
  Found Platform string: 'Nexus 6001 Chassis', Alias to: 'n6k'
  #### This device is of type: n6k #####
  
  ------------------------------------------------------------
  Section 1. Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.2 Default Properties exclusive to N6K and N7K [ensure => present]
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST    
  1.2 Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K :: RESOURCE    
    1.2 Default Properties exclusive to N6K and N7K :: RESOURCE     :: PASS
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K :: IDEMPOTENCE 
    1.2 Default Properties exclusive to N6K and N7K :: IDEMPOTENCE  :: PASS
  
  1.3 Default Properties exclusive to N7K :: default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 2. Non Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.2 Non Default Properties exclusive to N6K and N7K [ensure => present]
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST    
  2.2 Non Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K :: RESOURCE    
    2.2 Non Default Properties exclusive to N6K and N7K :: RESOURCE     :: PASS
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K :: IDEMPOTENCE 
    2.2 Non Default Properties exclusive to N6K and N7K :: IDEMPOTENCE  :: PASS
  
  2.3 Non Default Properties exclusive to N7K :: non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 3. vPC+ Non Default Property Testing
  
  3.1 vPC+ Non Default Properties on N7K :: vpc_plus_non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  3.1 vPC+ Non Default Properties on N7K :: vpc_plus_non_default_properties_n7k :: SKIP
  Platform type does not match testcase platform regexp: /n7k/
  
  ------------------------------------------------------------
    SKIPPED TESTS SUMMARY
  ------------------------------------------------------------
  1.3 Default Properties exclusive to N7K  :: SKIP
  2.3 Non Default Properties exclusive to N7K :: SKIP
  3.1 vPC+ Non Default Properties on N7K   :: SKIP
  3.1 vPC+ Non Default Properties on N7K   :: SKIP
  
  
  TestCase :: cisco_vpc_domain :: SKIP
  Warning: cisco_vpc_domain/test_vpc_domain.rb skipped in 672.94 seconds
      Test Suite: tests @ 2016-03-03 12:33:39 -0800

      - Host Configuration Summary -


              - Test Case Summary for suite 'tests' -
       Total Suite Time: 672.94 seconds
      Average Test Time: 672.94 seconds
              Attempted: 1
                 Passed: 0
                 Failed: 0
                Errored: 0
                Skipped: 1
                Pending: 0
                  Total: 1

      - Specific Test Case Status -
        
Failed Tests Cases:
Errored Tests Cases:
Skipped Tests Cases:
    Test Case cisco_vpc_domain/test_vpc_domain.rb skip
Pending Tests Cases:

  No tests to run for suite 'post_suite'
  Cleanup: cleaning up after successful run
  Warning: ssh connection to n6k-agent has been terminated
  Warning: ssh connection to puppetmaster-dc1 has been terminated
Beaker completed successfully, thanks.

N7K:
====
Begin cisco_vpc_domain/test_vpc_domain.rb

TestCase :: cisco_vpc_domain
  
  * 
  --------
   * TestStep :: Setup for cisco_vpc_domain provider test
  
  Found Platform string: 'Nexus7000 C7010 (10 Slot) Chassis', Alias to: 'n7k'
  #### This device is of type: n7k #####
  
  ------------------------------------------------------------
  Section 1. Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.2 Default Properties exclusive to N6K and N7K [ensure => present]
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST    
  1.2 Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K :: RESOURCE    
    1.2 Default Properties exclusive to N6K and N7K :: RESOURCE     :: PASS
  
  * TestStep :: 1.2 Default Properties exclusive to N6K and N7K :: IDEMPOTENCE 
    1.2 Default Properties exclusive to N6K and N7K :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.3 Default Properties exclusive to N7K [ensure => present]
  
  * TestStep :: 1.3 Default Properties exclusive to N7K [ensure => present] :: MANIFEST    
  1.3 Default Properties exclusive to N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 1.3 Default Properties exclusive to N7K :: RESOURCE    
    1.3 Default Properties exclusive to N7K :: RESOURCE     :: PASS
  
  * TestStep :: 1.3 Default Properties exclusive to N7K :: IDEMPOTENCE 
    1.3 Default Properties exclusive to N7K :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  1.1 Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  1.1 Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: RESOURCE    
    1.1 Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    1.1 Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 2. Non Default Property Testing
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => present]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.2 Non Default Properties exclusive to N6K and N7K [ensure => present]
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST    
  2.2 Non Default Properties exclusive to N6K and N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K :: RESOURCE    
    2.2 Non Default Properties exclusive to N6K and N7K :: RESOURCE     :: PASS
  
  * TestStep :: 2.2 Non Default Properties exclusive to N6K and N7K :: IDEMPOTENCE 
    2.2 Non Default Properties exclusive to N6K and N7K :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.3 Non Default Properties exclusive to N7K [ensure => present]
  
  * TestStep :: 2.3 Non Default Properties exclusive to N7K [ensure => present] :: MANIFEST    
  2.3 Non Default Properties exclusive to N7K [ensure => present] :: MANIFEST     :: PASS
  
  * TestStep :: 2.3 Non Default Properties exclusive to N7K :: RESOURCE    
    2.3 Non Default Properties exclusive to N7K :: RESOURCE     :: PASS
  
  * TestStep :: 2.3 Non Default Properties exclusive to N7K :: IDEMPOTENCE 
    2.3 Non Default Properties exclusive to N7K :: IDEMPOTENCE  :: PASS
  
  * 
  --------
  Create resource command title string:
    [:resource_name] 'cisco_vpc_domain'
    [:title_pattern] '200'
    
    title_string: '200'
  
  --------
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent]
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST    
  2.1 Non Default Properties on All Nexus Platforms [ensure => absent] :: MANIFEST     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: RESOURCE    
    2.1 Non Default Properties on All Nexus Platforms :: RESOURCE     :: PASS
  
  * TestStep :: 2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE 
    2.1 Non Default Properties on All Nexus Platforms :: IDEMPOTENCE  :: PASS
  
  ------------------------------------------------------------
  Section 3. vPC+ Non Default Property Testing
  
  * Check for Compatible Line Module
    Test interface name is 'ethernet1/1'
    
    * Get default VDC name
      
      * Set 'limit-resource module-type f2e f3'
        
        * Verify that 'ethernet1/1' is allocated to VDC
          
          * 
          --------
          Create resource command title string:
            [:resource_name] 'cisco_vpc_domain'
            [:title_pattern] '200'
            
            title_string: '200'
          
          --------
          3.1 vPC+ Non Default Properties on N7K [ensure => present]
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K [ensure => present] :: MANIFEST    
          3.1 vPC+ Non Default Properties on N7K [ensure => present] :: MANIFEST     :: PASS
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K :: RESOURCE    
            3.1 vPC+ Non Default Properties on N7K :: RESOURCE     :: PASS
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K :: IDEMPOTENCE 
            3.1 vPC+ Non Default Properties on N7K :: IDEMPOTENCE  :: PASS
          
          * 
          --------
          Create resource command title string:
            [:resource_name] 'cisco_vpc_domain'
            [:title_pattern] '200'
            
            title_string: '200'
          
          --------
          3.1 vPC+ Non Default Properties on N7K [ensure => absent]
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K [ensure => absent] :: MANIFEST    
          3.1 vPC+ Non Default Properties on N7K [ensure => absent] :: MANIFEST     :: PASS
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K :: RESOURCE    
            3.1 vPC+ Non Default Properties on N7K :: RESOURCE     :: PASS
          
          * TestStep :: 3.1 vPC+ Non Default Properties on N7K :: IDEMPOTENCE 
            3.1 vPC+ Non Default Properties on N7K :: IDEMPOTENCE  :: PASS
        TestCase :: Resource cisco_vpc_domain :: End
        cisco_vpc_domain/test_vpc_domain.rb passed in 947.98 seconds
      Test Suite: tests @ 2016-03-03 13:19:18 -0800

      - Host Configuration Summary -


              - Test Case Summary for suite 'tests' -
       Total Suite Time: 947.98 seconds
      Average Test Time: 947.98 seconds
              Attempted: 1
                 Passed: 1
                 Failed: 0
                Errored: 0
                Skipped: 0
                Pending: 0
                  Total: 1

      - Specific Test Case Status -
        
Failed Tests Cases:
Errored Tests Cases:
Skipped Tests Cases:
Pending Tests Cases:

        No tests to run for suite 'post_suite'
        Cleanup: cleaning up after successful run
        Warning: ssh connection to puppetagent-n7k9 has been terminated
        Warning: ssh connection to puppetmaster-dc1 has been terminated
Beaker completed successfully, thanks.

File: README.md [hunk 1]
```diff
@@ -2738,6 +2750,10 @@ Enable or Disable Layer3 forwarding for packets with peer gateway-mac. Valid val
 ##### `peer_gateway_exclude_vlan`
 Interface vlans to exclude from peer gateway functionality. Valid value is a string of integer ranges from 1..4095. This parameter is available only in Nexus 5000, Nexus 6000 and Nexus 7000 series. There is no default value.
 
+##### `port_channel_limit`
+In vPC+ mode, enable or disable the port channel scale limit of
+244 vPCs.  Valid values are true, false or default
```
Comment (mikewiebe) [inline README.md]: Is there risk of this number changing on one of the platforms in the future?

File: examples/cisco/demo_vpc_plus.pp [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# Manifest to demo cisco_vpc_domain provider
+#
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
```
Comment (mikewiebe) [inline examples/cisco/demo_vpc_plus.pp]: `2016`

File: tests/beaker_tests/cisco_vpc_domain/test_vpc_domain.rb [hunk 3]
```diff
@@ -73,8 +73,10 @@
 # tests[:show_cmd] - the common show command to use for test_show_run
 #
 tests = {
```
Comment (mikewiebe) [inline tests/beaker_tests/cisco_vpc_domain/test_vpc_domain.rb]: Please add any platform checks needed to skip properties that are not suported on `n8k`

File: README.md (hunk 1 above)
Comment (dcheriancisco) [inline README.md]: No , this cannot change as it is limited by the protocol field size of 8 bits (sub-switch-id). This exists only in N7K (with F2 and lower rev modules). In other platforms, there is no option to use sub-switch-id and so there is no such limit. They use a second lookup on the mac-table to determine the port of exit.

**Pull request merged**